agent/fw_wordpress.c

```diff
@@ -735,6 +745,11 @@ void nr_wordpress_enable(TSRMLS_D) {
       nr_wordpress_handle_tag_stack_after, nr_wordpress_handle_tag_stack_clean,
       options);
 
+  nr_php_wrap_user_function(NR_PSTR("add_action"),
+                            nr_wordpress_add_action_filter);
+  nr_php_wrap_user_function(NR_PSTR("add_filter"),
+                            nr_wordpress_add_action_filter);
```

Comment:
`add_action` is a wrapper around `add_filter` (quote from WordPress code base):
```
function add_action($tag, $function_to_add, $priority = 10, $accepted_args = 1) {
	add_filter($tag, $function_to_add, $priority, $accepted_args);
}
```
Wouldn't instrumenting `add_filter` be enough?

Reply:
Are `add_action` and `add_filter` the only functions that register user callbacks for various WordPress hooks?

Reply:
Good point! I have done a manual audit of the WP source code (and a quick internet search) and do believe that there is no other callback to use besides `add_filter`. I will remove `add_action` instrumentation and change the tests.

Reply:
fixed in [28480e6](https://github.com/newrelic/newrelic-php-agent/pull/731/commits/28480e616ba88f418f171de4e5a1de8a9116f915)